c7n/resources/fis.py

```diff
@@ -87,3 +87,54 @@ def process(self, resources):
         client = local_session(self.manager.session_factory).client('fis')
         for r in resources:
             self.manager.retry(client.delete_experiment_template, id=r['id'])
+
+
+class ExperimentDescribe(DescribeSource):
+    def augment(self, resources):
+        resources = super().augment(resources)
+        # tag normalize for value filter
+        for r in resources:
+            if 'tags' not in r:
+                continue
+            r['Tags'] = [{'Key': k, 'Value': v} for k, v in r.pop('tags', {}).items()]
+        return resources
+
+
+@resources.register('fis-experiment')
+class Experiment(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = 'fis'
+        enum_spec = ('list_experiments', 'experiments', None)
+        detail_spec = ('get_experiment', 'id', 'id', 'experiment')
+        name = id = 'id'
+        date = 'creationTime'
+
+    source_mapping = {'describe': ExperimentDescribe,
+                      'config': ConfigSource}
```

Comment:
experiments aren't supported by config so can omit that source, only experiment templates are supported.